### PR TITLE
[TASK] Update for TYPO3 10 LTS compatibility

### PR DESCRIPTION
**Pull request by liayn**
Partial first fix for #30

### CODE REVIEW

**Comment by liayn** [comment]
@Tuurlijk I did some more stuff.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.idea/
```
Comment (Tuurlijk) [inline .gitignore:1]: Why though?

Comment (liayn) [reply .gitignore:1]: Because we don't want anybody to commit the phpstorm project config?

Comment (Tuurlijk) [reply .gitignore:1]: Ah, I missed that. Did not see that the file was .gitignore. Assumed that it was a committed dir . . . oh well  . . Assumption is the mother of all f*ckups :-)

**Pull request merged**